packages/diff-sequences/src/__tests__/index.property.test.ts

```diff
@@ -0,0 +1,82 @@
+import diff from '../';
+import fc from 'fast-check'
+
+const findCommonItems = (a: string[], b: string[]): string[] => {
+  const array: string[] = [];
+  diff(
+    a.length,
+    b.length,
+    (aIndex: number, bIndex: number) => a[aIndex] === b[bIndex],
+    (nCommon: number, aCommon: number, bCommon: number) => {
+      for (; nCommon !== 0; nCommon -= 1, aCommon += 1) {
+        array.push(a[aCommon]);
+      }
+    },
+  );
+  return array;
+};
+
+const flatten = (data: string[][]) => {
+  const array: string[] = [];
+  for (const items of data) {
+    array.push(...items)
+  }
+  return array;
+}
+
+test('should be reflexive', () => {

```

Comment:
You have written the test names so that they read fluently with `it` instead of `test` blocks

Reply:
Do you prefer that I switch back to test names for `test()`?
Just to be sure: If I move to `it()`, do I need to put them under a `describe()` too?

Reply:
Jest does not require `describe` block for `it` and use whichever expresses your meaning.